style.css

```diff
@@ -296,3 +296,111 @@ a {
   position: relative;
   margin-bottom: 16px;
 }
+
+.section-bg {
+  padding-top: 68px;
+  padding-bottom: 68px;
+  padding-left: 16px;
+  padding-right: 16px;
+  background-color: #f9f9fa;
+  background-image: url(./assets/geo.png);
+  background-repeat: no-repeat;
+}
```

Comment:
- [ ] Kindly make sure to layout the form contents by using `flex-box` as requested in the project requirements.